src/screens/Accounts/Accounts.js

```diff
@@ -224,69 +230,80 @@ class AccountsScreen extends React.Component<Props, State> {
     navigation.navigate(ASSETS);
   };
 
-  renderNetworks = ({ item }: Object) => {
+  renderPPN(ppnNetwork: BlockchainNetwork, isActive: boolean) {
     const {
       navigation,
-      baseFiatCurrency,
       availableStake,
       isTankInitialised,
-      smartWalletFeatureEnabled,
-      blockchainNetworks,
     } = this.props;
 
-    const {
-      type = '',
-      balance = {},
-      id,
-      action,
-    } = item;
+    const availableStakeFormattedAmount = formatMoney(availableStake, 4);
+
+    return (
+      <ListCard
+        {...ppnNetwork}
+        subtitle={`Balance: ${availableStakeFormattedAmount} ${PPN_TOKEN}`}
+        action={isTankInitialised
+          ? this.setPPNAsActiveNetwork
+          : () => navigation.navigate(PILLAR_NETWORK_INTRO)}
+        note={{
+          note: 'Instant, free and private transactions',
+          emoji: 'sunglasses',
+        }}
+        iconSource={pillarNetworkIcon}
+        contentWrapperStyle={{
+          borderWidth: 2,
+          borderColor: isActive ? baseColors.electricBlue : baseColors.white,
+          borderRadius: 6,
+        }}
+      >
+        {isActive && <CheckIcon name="check" />}
+      </ListCard>
+    );
+  }
+
+  renderNetwork(item: NetworkItem) {
+    const { network } = item;
+    const { id } = network;
+
+    const { blockchainNetworks } = this.props;
 
-    const activeBNetwork = blockchainNetworks.find((network) => network.isActive) || { id: '' };
-    const { id: activeBNetworkID } = activeBNetwork;
+    const activeNetwork = blockchainNetworks.find((net) => net.isActive);
+    const isActive = !!activeNetwork && activeNetwork.id === id;
 
     if (id === BLOCKCHAIN_NETWORK_TYPES.PILLAR_NETWORK) {
-      const availableStakeFormattedAmount = formatMoney(availableStake, 4);
-
-      if (!smartWalletFeatureEnabled) return null;
-      return (
-        <ListCard
-          {...item}
-          subtitle={`Balance: ${availableStakeFormattedAmount} ${PPN_TOKEN}`}
-          action={isTankInitialised
-            ? this.setPPNAsActiveNetwork
-            : () => navigation.navigate(PILLAR_NETWORK_INTRO)}
-          note={{
-            note: 'Instant, free and private transactions',
-            emoji: 'sunglasses',
-          }}
-          iconSource={pillarNetworkIcon}
-          contentWrapperStyle={{
-            borderWidth: 2,
-            borderColor: activeBNetworkID === BLOCKCHAIN_NETWORK_TYPES.PILLAR_NETWORK
-              ? baseColors.electricBlue
-              : baseColors.white,
-            borderRadius: 6,
-          }}
-        >
-          {activeBNetworkID === BLOCKCHAIN_NETWORK_TYPES.PILLAR_NETWORK && <CheckIcon name="check" />}
-        </ListCard>
-      );
+      return this.renderPPN(network, isActive);
```

Comment:
Considering that cards for Networks will have the same layout, we could simply reuse `renderPPN` method (renamed, of course) to render all other networks. Checking network type to call render method can result in duplicate code for card rendering :)